qutebrowser/misc/sessions.py

```diff
@@ -342,6 +347,13 @@ def _load_tab(self, new_tab, data):
             itertools.dropwhile(lambda i: i[0] < lazy_index,
                                 enumerate(data['history'])))
 
+        if config.val.tabs.mode_on_change == 'restore':
+            if 'input_mode' in data:
+                input_mode = usertypes.KeyMode(data['input_mode'])
+            else:
+                input_mode = usertypes.KeyMode.normal
```

Comment:
When using strings, you can use `data.get('input_mode', 'normal')` here instead.

Reply:
Ok, but this is not a string, it's an enum.

Reply:
But you're right, I can do this:

```
            input_mode = data.get('input_mode', usertypes.KeyMode.normal.value)
            new_tab.data.input_mode = usertypes.KeyMode(input_mode)
```

Reply:
But it *should* be a string, like I said above...